external-api/src/types/api_order_book.rs

```diff
@@ -44,3 +44,19 @@ impl From<NetworkOrder> for ApiNetworkOrder {
         }
     }
 }
+
+/// A token in the the supported token list
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct ApiToken {
+    /// The token address
+    pub address: String,
+    /// The token symbol
+    pub symbol: String,
```

Comment:
Should this be `ticker` for consistency with the token map?

Reply:
symbol is better on the api layer IMO b/c it is more common in other interfaces (e.g. the ERC20 abi)